.eslintrc

```diff
@@ -7,30 +7,45 @@
     "mocha": true
   },
   "rules": {
-    "no-console": 0,
+    "strict": 0,
     "quotes": 0,
     "one-var": 0,
+    "no-console": 0,
+    "no-unused-vars": 0,
+    "implicit-arrow-linebreak": 0,
+    "nes-around-directive": 0,
+    "import/no-dynamic-require": 0,
+    "prefer-template": 0,
     "one-var-declaration-per-line": 0,
+    "import/newline-after-import": 0,
+    "class-methods-use-this": 0,
     "new-cap": 0,
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 0,
-    "arrow-parens": ["error", "always"],
+    "arrow-parens": [0, "error", "always"],
```

Comment:
I think we must adhere to the same rules.

Reply:
Ooooh... I forgot to undo the changes